lib/Twig/Loader/Filesystem.php

```diff
@@ -150,9 +150,44 @@ public function exists($name)
             return true;
         }
 
-        if (isset($this->errorCache[$name])) {
+        try {
+            return false !== $this->findTemplate($name, false);
+        } catch (Twig_Error_Loader $exception) {
```

Comment:
The catch here is for implementation which do not respect the `$throw` argument.